<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustAdd</class>
 <widget class="QDialog" name="CustAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Customer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SosSecNr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="FirstName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>SosSecNR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>LastName</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Phone Nr</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>FirstName
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddCust">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
